environments/openstack/playbook-bootstrap-basic.yml

```diff
@@ -203,10 +211,23 @@
         disk_format: qcow2
         filename: /tmp/ubuntu.img
         min_disk: 3
+        min_ram: 512
         properties:
           cpu_arch: x86_64
           distro: ubuntu
           hw_rng_model: virtio
+          architecture: x86_64
+          hypervisor_type: qemu
+          os_version: "20.04"
+          os_distro: ubuntu
+          hw_disk_bus: virtio
+          replace_frequency: never
```

Comment:
Of course, I would expect a serious cloud provider to have at least `monthly` here.
But not setting it by default is probably reasonable, as this way providers can not plead innocent for having claimed monthly updates because they have overlooked changing the default ...

Reply:
In the case of the testbed, the image is not currently being updated and the entry is therefore correct.